### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by lostest4352** [tomgilder/routemaster#334]
Is this package dead? Since gorouter has seemingly turned out bad from what I read on twitter and discords with poor design decisions(I'm not skilled enough to judge that but see this opinion a lot without having any disagreement replies), I hope this would be continued to developed. Other options I looked doesnt seem to be user friendly and approachable as routemaster is. Its simple yet does everything that I need and with clear documentation.

**Comment by killik-dan-stubbs** [comment]
I'd say that even though this package hasn't been updated in the last two years, it still works extremely well and should do for years to come. More frequent updates would be desirable to make sure the `pubspec.yaml` is updated to the latest dependencies, for now it works fine! (Flutter 3.22.1)

**Comment by martipello** [comment]
i would argue this packsge is dead and you should migrate to go router or similar

**Comment by killik-dan-stubbs** [comment]
Yeah, after more consideration, this should probably be considered dead. I have a new preference for using [auto_route](https://pub.dev/packages/auto_route) over [go_router](https://pub.dev/packages/go_router) as it solves some of the problems (mostly observation) issues I have had with go_router. Give each a shot though, they're great. Don't make the same mistake I made in using this package, you may well end up regretting it as I did.
